src/Executor/Executor.php

```diff
@@ -24,19 +24,13 @@ class Executor
 
     protected array $headers;
 
-    protected int $cpus;
-
-    protected int $memory;
-
     public function __construct(string $endpoint)
     {
         if (!filter_var($endpoint, FILTER_VALIDATE_URL)) {
             throw new Exception('Unsupported endpoint');
         }
 
         $this->endpoint = $endpoint;
-        $this->cpus = \intval(System::getEnv('_APP_FUNCTIONS_CPUS', '1'));
-        $this->memory = \intval(System::getEnv('_APP_FUNCTIONS_MEMORY', '512'));
```

Comment:
are we deprecating these variables now? Are they used anywhere?

Reply:
No, these are now enforced as limits within the validators